src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets

```diff
@@ -56,6 +56,16 @@ Copyright (c) .NET Foundation. All rights reserved.
           DependsOnTargets="Pack">
   </Target>
 
+  <Target Name="_CleanPackageFiles"
+          AfterTargets="Clean"
```

Comment:
As a rule, I prefer and recommend using `AfterTargets`/`BeforeTargets` rather than the `*DependsOn` properties. There are situations (like the other thing I've been talking to @rohit21agrawal about) that mean that the properties are better, but I find before/after easier to reason about and debug.